azure-pipelines.yml

```diff
@@ -0,0 +1,82 @@
+parameters:
+  - name: terraformVersion
+    type: string
+    default: "0.13.4"
```

Comment:
this shouldn't be a parameter it should be a variable, and should be using 0.14.x

Reply:
fixed.